riscv/cva5/cva5_tapasco.patch

```diff
@@ -11,10 +11,9 @@ index c7d5550..19d71d5 100755
 +        INCLUDE_S_MODE : 0,
 +        INCLUDE_U_MODE : 0,
          INCLUDE_MUL : 1,
--        INCLUDE_DIV : 1,
```

Comment:
Please keep the configuration the same. Divider is not needed in most of the tapasco-riscv use cases.

Reply:
Done!